analytical_engine/CMakeLists.txt

```diff
@@ -457,6 +483,7 @@ endmacro()
 install_gsa_binary(grape_engine)
 install_gsa_binary(gs_proto)
 install_gsa_binary(gs_util)
+install_gsa_binary(graphx_runner)
```

Comment:
The target `graphx_runner` only exists when `ENABLE_JAVA_SDK=ON`.

Missing the check leads to error in https://github.com/alibaba/GraphScope/actions/runs/3100414445/jobs/5020669581